torch/_dynamo/utils.py

```diff
@@ -774,6 +774,11 @@ def torch_clone(x):
             # Access data_ptr() for a xla tensor will cause crash
             return torch_clone(x)
 
+        if x.layout is not torch.strided:
```

Comment:
We should explicitly check that layout is one of the sparse layouts. There exists other layouts than sparse and strided so that `x.layout is not torch.strided` is not equivalent to "x is a sparse COO/CSR/CSC/BSR/BSC tensor".